SonarKit.podspec

```diff
@@ -13,6 +13,7 @@ Pod::Spec.new do |spec|
   spec.source = { :git => 'https://github.com/facebook/Sonar.git',
                   :branch=> "master" }
   spec.module_name = 'SonarKit'
+  spec.swift_version = '4.1'
```

Comment:
This option is only available in CocoaPods >= 1.4, you might want to add another line to the spec:
```ruby
spec.cocoapods_version = '>= 1.4.0'  
```